test/xref2/references_to_assets.t/run.t

```diff
@@ -0,0 +1,71 @@
+In this file, we test the resolving of asset references.
```

Comment:
Nothing in this test shows a working asset reference. Shouldn't `caml.gif_hidden` be renamed ?
Also, the only reference that seems to be intended to work do not test page parents (there's no `other_page."caml.gif"`).

Reply:
Yes, at the end there are working asset references. The assets are not passed in the html generation phase, which creates "dead links" just as there are dead links when you do not html-generate some of the pages), but the test checks that the links point to the right place.
Do you think it would be better to actually pass some image in the html generation, to compare the path more accurately?

Reply:
I went ahead and passed assets in the test.